airflow/www/templates/airflow/dag_code.html

```diff
@@ -21,6 +21,14 @@
 
 {% block page_title %}{{ dag.dag_id }} - Code - Airflow{% endblock %}
 
+{% block head %}
+  {{ super() }}
+  <head>
```

Comment:
This will create duplicate `<head>` tags won't it?

Reply:
Still need to remove these tags

Reply:
of course. my bad.